home/cachyos-performance.md

```diff
@@ -8,11 +8,11 @@ editor: markdown
 dateCreated: 2021-10-04T07:59:10.433Z
 ---
 
-# Linux Kernel with BORE scheduler is the default choice
+# Linux Kernel with EEDVF scheduler as the default scheduler
 
-Yes, this is a part of the optimization for CachyOS. ¿Why do we use that scheduler in particular?. Well... let us explain why!. There are so many schedulers like BMQ, MuQSS, that some of you might already know (or not), but we have tested BORE and it resulted the one with the best performance results both in benchmarks and responsiveness tests.
+An important aspect of optimization for CachyOS is the scheduler used in our default kernel. Let us explain why we have chosen the EEVDF as our default scheduler. There are many schedulers available, like BMQ or MuQSS, some of them you might already know. We have tested EEVDF and it resulted the one with the best performance results both in benchmarks and responsiveness tests. Please keep an eye out for any updates, as schedulers tend to change often and our default scheduler might change in the future.
```

Comment:
Can you please move out MuQSS and BMQ?
I don't think this should be in there. MuQSS is obsolete at all since more then a year.

Reply:
Replaced with BORE and PDS, which are more known.